crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -87,6 +193,8 @@ pub(crate) fn removed_in_3(checker: &mut Checker, expr: &Expr) {
                 check_call_arguments(checker, &qualname, arguments);
             };
             check_method(checker, call_expr);
+            check_removed_context_keys_usage(checker, call_expr);
+            check_removed_context_keys_get_anywhere(checker, call_expr);
```

Comment:
This doesn't really make sense as both function is going to perform the same check except that the first one is behind an additional check of `@task` function. I'm not sure why this is present.